css/style.css

```diff
@@ -263,36 +263,27 @@ input[type="checkbox"] {
 /* ----------- UI - Graph ----------- */
 
 .vis {
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  padding: 30px;
-}
-
-.vis > div {
-  display: flex;
-  align-items: flex-start;
-  justify-content: space-between;
-  height: 100%;
+  display: grid;
+  grid-template-rows: 75% 25%;
+  position: relative;
 }
 
-.vis > div > div {
-  display: flex;
-  flex-direction: column;
-  height: 100%;
+.vis-panel {
+  padding: 30px;
+  padding-bottom: 0;
```

Comment:
Per my [comment](https://github.com/electrolyfish/lightbeam-we/pull/112#issuecomment-318718783) on the PR, reverting `.vis-panel` is part 2 of 3 of the fix. The divs are not exactly the same here since the HTML was restructured, but it's a combination of the two blocks that were removed:

```css
.vis-panel {
  display: flex;
  flex-direction: column;
  height: 100%;
}
```